evaluation/swe_bench/run_infer.py

```diff
@@ -39,11 +39,14 @@
 AGENT_CLS_TO_FAKE_USER_RESPONSE_FN = {
     'CodeActAgent': codeact_user_response,
     'CodeActSWEAgent': codeact_user_response,
+    'CoActPlannerAgent': codeact_user_response,
 }
 
+codeact_inst_suffix = 'When you think you have fixed the issue through code changes, please run the following command: <execute_bash> exit </execute_bash>.\n'
```

Comment:
Remove spaces around `exit`?

Reply:
It's from the original CodeAct eval, I guess it's fine to leave it as is. In the action parser we'll strip it anyway.